integration-tests/src/tests/mod.rs

```diff
@@ -8,3 +8,5 @@ mod test_errors;
 mod test_overflows;
 mod test_simple;
 mod test_tps_regression;
+mod test_helpers;
+mod genesis_helpers;
```

Comment:
moved test_helpers and genesis_helpers from `integration-tests/` to `integration-tests/tests/`